integreat_cms/cms/templates/_base.html

```diff
@@ -67,7 +67,7 @@
             </div>
         </div>
         <div id="user-info"
-             class="relative px-2 text-gray-800 flex flex-col justify-center cursor-pointer hover:bg-gray-200">
+             class="hidden relative px-2 text-gray-800 lg:flex flex-col justify-center cursor-pointer hover:bg-gray-200">
```

Comment:
Just hiding this removes two essential functions: Accessing the user settings and logout.
I like @david-venhoff's idea of hiding some of the text and removing the link to make it possible to tap on the icon to expand the menu.